src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type ControllerResponse = {
+  isSuccess: number;
+  statusCode: number;
+  message: string;
+  data?: any;
+  error?: any;
+};
+
+export const APIResponse = {
+  create: (
+    isSuccess: number,
+    statusCode: number,
+    message?: string,
+    data?: any,
+    error?: any,
+  ): ControllerResponse => {
```

Comment:
PrimathonAICodeReviewer

Consider using TypeScript interfaces instead of type aliases for better readability and maintainability.
```diff
- export type ControllerResponse = {
+ export interface ControllerResponse {
```
```diff
- export const APIResponse = {
+ export const APIResponse: {
```



<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->